var/spack/repos/builtin/packages/graphviz/package.py

```diff
@@ -93,32 +103,34 @@ def configure_args(self):
         spec = self.spec
         options = []
 
-        # These language bindings have been tested, we know they work.
-        tested_bindings = ('+java', '+perl')
-
-        # These language bindings have not yet been tested.  They
-        # likely need additional dependencies to get working.
-        untested_bindings = (
-            '+swig', '+sharp', '+go', '+guile', '+io',
-            '+lua', '+ocaml', '+php',
-            '+python', '+r', '+ruby', '+tcl')
+        need_swig = False
 
-        for var in untested_bindings:
+        for var in self.untested_bindings:
             if var in spec:
-                raise SpackException(
+                raise InstallError(
                     "The variant {0} for language bindings has not been "
                     "tested.  It might or might not work.  To try it "
                     "out, run `spack edit graphviz`, and then move '{0}' "
                     "from the `untested_bindings` list to the "
                     "`tested_bindings` list.  Be prepared to add "
                     "required dependencies.  "
                     "Please then submit a pull request to "
-                    "http://github.com/llnl/spack")
+                    "http://github.com/llnl/spack".format(var))
+            options.append('--disable-%s' % var[1:])
 
-        for var in tested_bindings:
-            enable = 'enable' if (var in spec) else 'disable'
+        for var in self.tested_bindings:
+            if (var in spec):
+                need_swig = True
+                enable = 'enable'
+            else:
+                enable = 'disable'
```

Comment:
I inherited that bit and/but am not particularly fond of it.

Switched it around to always use the `--enable-foo=(yes|no)` form of the configure arg so that it's consistent with other bits.  In this context `enable` is either *yes* or *no*, which seems ok.  Thought about naming the variable `enabled`, but I figured I should save the character in case I need it later.  ;)